code/modules/projectiles/guns/syringe_gun.dm

```diff
@@ -51,10 +52,12 @@
 	var/obj/item/reagent_containers/syringe/S
 	if(length(syringes))
 		S = syringes[length(syringes)]
+		S.mode = SYRINGE_INJECT
```

Comment:
This would be an unchambered one, most likely without any reagents? Should that also be ready to inject?

Reply:
Yeah I think this is for the RSG, which indeed only accepts empty ones, and fills the chambered one. Hence why this will make the empty syringes ready to be injected